internal/model/board.go

```diff
@@ -33,32 +33,107 @@ var (
 	players = []*Player{
 		{
 			Color: ColorBlue,
-			Line:  0,
-			Row:   0,
+			Position: &Coordinate{
+				Line: 0,
+				Row:  0,
+			},
 			Score: 0,
 		},
 		{
 			Color: ColorGreen,
-			Line:  -1,
-			Row:   -1,
+			Position: &Coordinate{
+				Line: -1,
+				Row:  -1,
+			},
 			Score: 0,
 		},
 		{
 			Color: ColorRed,
-			Line:  0,
-			Row:   -1,
+			Position: &Coordinate{
+				Line: 0,
+				Row:  -1,
+			},
 			Score: 0,
 		},
 		{
 			Color: ColorYellow,
-			Line:  -1,
-			Row:   0,
+			Position: &Coordinate{
+				Line: -1,
+				Row:  0,
+			},
 			Score: 0,
 		},
 	}
 	remainingPlayers = []int{0, 1, 2, 3}
 )
 
+// TileExit represents the possible exits that are available for the tile.
+type TileExit int
+
+const (
+	TileExitTop TileExit = iota
+	TileExitRight
+	TileExitBottom
+	TileExitLeft
+)
+
+func (t TileExit) Opposite() TileExit {
+	switch t {
+	case TileExitTop:
```

Comment:
This could also be a data structure:

```js
const oppositeDirections = {
  'top': 'bottom',
  'bottom': 'top',
  'left': 'right',
  'right': 'left',
};
const getOppositeDirection = direction => oppositeDirections[direction];
```